### PR TITLE
Align Helm Broker chart with external repo version

### PR DESCRIPTION
**Pull request by mszostok**
**Description**

Changes proposed in this pull request:

- Align Helm Broker chart with external repo version

**Related issue(s)**

- kyma-project/kyma#6644

### CODE REVIEW

File: resources/helm-broker/values.yaml [hunk 1]
```diff
@@ -20,24 +19,51 @@ ctrl:
...
+  helm_broker:
+    dir: /develop/
+    version: "e941b81f"
+  helm_controller:
+    dir: /develop/
+    version: "e941b81f"
+  # etcd port uses by helm broker
+  etcdClientPort: 2379
+
+  # Additional values for
```
Comment (polskikiel) [inline resources/helm-broker/values.yaml]: for what?

File: resources/cluster-essentials/templates/addons-configuration.crd.yaml [hunk 2]
```diff
@@ -2,9 +2,9 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   creationTimestamp: null
-  labels:
-    controller-tools.k8s.io: "1.0"
   name: addonsconfigurations.addons.kyma-project.io
+  annotations:
+    helm.sh/hook: crd-install
```
Comment (polskikiel) [inline resources/cluster-essentials/templates/addons-configuration.crd.yaml]: are you sure about this? other crds definitions don't have it

File: resources/helm-broker/templates/helm-secret.yaml [hunk 3]
```diff
@@ -7,3 +8,4 @@ metadata:
   name: helm-secret
   namespace: kyma-system
```
Comment (polskikiel) [inline resources/helm-broker/templates/helm-secret.yaml]: You can use `{{ Release.Namespace }}`

Comment (mszostok) [reply resources/helm-broker/templates/helm-secret.yaml]: nice catch, I've removed that entry totally

File: resources/cluster-essentials/templates/addons-configuration.crd.yaml (hunk 2 above)
Comment (mszostok) [reply resources/cluster-essentials/templates/addons-configuration.crd.yaml]: I saw that, and have that should not cause any problem but I can remove it

File: resources/helm-broker/values.yaml (hunk 1 above)
Comment (mszostok) [reply resources/helm-broker/values.yaml]: xD updated

**Pull request merged**